website/src/components/ReviewPage/ReviewCard.tsx

```diff
@@ -403,19 +403,31 @@ const KeyValueComponent: FC<KeyValueComponentProps> = ({
     const textTooltipId = 'text-tooltip-' + keyName + accessionVersion;
     const noteTooltipId = 'note-tooltip-' + keyName + accessionVersion;
 
+    const textRef = useRef<HTMLSpanElement>(null);
+    const [isTruncated, setIsTruncated] = useState(false);
+
+    useEffect(() => {
+        if (textRef.current) {
+            setIsTruncated(textRef.current.scrollWidth > textRef.current.clientWidth);
+        }
+    }, [value]);
```

Comment:
I'm not sure here but I am just wondering: do we need to add `textRef.current` as a dependency as well?

Reply:
References are kind of outside the React change-detection logic. This will just run immediately after render and that's ok here

Reply:
(thanks for checking!)